<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTrack</class>
 <widget class="QDialog" name="EditTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Track Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Extra Info</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="extInfo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="trackNumber">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="prevTrack">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextTrack">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>EditTrack</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
